index.html

```diff
@@ -5,6 +5,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Map</title>
     <script type="module" src="/src/my-map.ts"></script>
+    <!-- OS vector tile source specifies fonts in .pbf format, which OpenLayers can't load, so make them available directly  -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;600&display=swap" rel="stylesheet">
```

Comment:
This will seem pretty weird but I think these CSS imports can go inside the html block of the render() function! Then it's all a self-contained component.

In future we could also make them conditionally load depending on whether you're loading a vector or raster layer.

Reply:
ah yeah this is interesting! 

not quite as simple as just putting the links in render() on first try, but found [this thread](https://stackoverflow.com/questions/57489637/how-to-load-google-font-in-litelement) which seems to have some good clues. 

think i'll leave links temporarily in index.html for this merge, and come back to later!